demo/README.md

```diff
@@ -35,7 +35,13 @@ Debug config can be like:
 }
 ```
 
-# Debugging test with chrome devtools
+*In case of windows `node_modules/.bin` folder contains executable files
+ that are not javaScript programs so they cannot be launched with `node`
+ interpreter instead one can use direct path to javaScript file, e.g. 
+ `node_modules/suitest-js-api/bin/suitest` and in case of 
```

Comment:
@AntonSBilan @givehug @andriichumak  If we have have configuration which works on all platforms, maybe better use it and dont separate guide for different platforms?

Reply:
As for me I prefer to use files in ./bin folder, no need to look for correct path in package.json but @andriichumak  has it sense to make separate section for windows users with example configs in readme and user docs or these who use js api are rather advanced to know about their platform specific issues?

Reply:
I think it's enough as long as we account for things that user can't fix on his end. E.g. make sure that **all** string parameters are wrapped with quotes when needed (i.e. when they hold spaces).